elife/masterless.sls

```diff
@@ -14,7 +14,11 @@ disable-salt-minion:
 # zero out all allowed keys
 deny_all_access:
     cmd.run:
-        - name: echo > /home/elife/.ssh/authorized_keys
+        - name: |
+            set -e
+            echo > /home/elife/.ssh/authorized_keys
+            cat /home/ubuntu/.ssh/authorized_keys | grep -E "{{ salt['elife.cfg']('project.stackname') }}|basebox" > tmp
```

Comment:
not sure what the `grep` is preserving?

Reply:
ah, there are two keys in /home/ubuntu/.ssh/authorized_keys that should probably be preserved. a pubkey with the name of the stack, like `lax--prod`, probably put there automatically by the ami/ec2 creation system so we can login. The other is a key called 'basebox' that I'm not sure about - I included it thinking it might solve the build error we're seeing here but that wasn't the case. The 'basebox' key might be the pubkey from the original ami, in which case it could probably be removed, but I thought you might be depending on it for some reason.